### PR TITLE
[RW-3754][Risk=no] Remove getWorkspace from create

### PR DESCRIPTION
**Pull request by blrubenstein**

### CODE REVIEW

**Comment by blrubenstein** [comment]
Already done! It appears to work fine and creates a new workspace.

**Comment by jaycarlton** [comment]
@blrubenstein if you don't mind, next time it would help on non-trivial PRs to put a description discussing what the problem was, any context around it, what the solution is, and any testing done.

File: api/src/main/java/org/pmiops/workbench/firecloud/FireCloudServiceImpl.java [hunk 1]
```diff
@@ -290,17 +291,13 @@ public void removeOwnerFromBillingProject(
...
     WorkspaceIngest workspaceIngest = new WorkspaceIngest();
     workspaceIngest.setName(workspaceName);
     workspaceIngest.setNamespace(projectName);
     checkAndAddRegistered(workspaceIngest);
-    retryHandler.run(
-        (context) -> {
-          workspacesApi.createWorkspace(workspaceIngest);
-          return null;
-        });
+    return retryHandler.run((context) -> workspacesApi.createWorkspace(workspaceIngest));
```
Comment (jaycarlton) [inline api/src/main/java/org/pmiops/workbench/firecloud/FireCloudServiceImpl.java]: maybe name your lambda parameter `unusedContext` or similar.

**Comment by blrubenstein** [comment]
Sure, I can throw that context on there in the future. Although I'd rather details around problem, context, and solution be on the ticket (which I also made, and could have put more details on). In this case, there was an extra get call that wasn't necessary which was skewing our perf results. This involved removing that get call, since the create call had all the information we needed in the response.

**Comment by jaycarlton** [comment]
👍 For most PRs I don't go back to look up the ticket, since it's usually a discussion of the problem only. Also our github is public but Jira is private, fwiw.

**Pull request merged**